fpdf/fpdf.py

```diff
@@ -3694,47 +3705,91 @@ def write(
             styled_text_fragments,
+            lambda h: max_width,
+            (self.c_margin, self.c_margin),
             print_sh=print_sh,
             wrapmode=wrapmode,
         )
         # first line from current x position to right margin
         first_width = self.w - self.x - self.r_margin
-        txt_line = multi_line_break.get_line_of_given_width(
-            first_width - 2 * self.c_margin,
-        )
+        max_width = first_width
+        text_line = multi_line_break.get_line()
         # remaining lines fill between margins
         full_width = self.w - self.l_margin - self.r_margin
-        fit_width = full_width - 2 * self.c_margin
-        while txt_line is not None:
-            text_lines.append(txt_line)
-            txt_line = multi_line_break.get_line_of_given_width(fit_width)
+        max_width = full_width
+        while (text_line) is not None:
+            text_lines.append(text_line)
+            text_line = multi_line_break.get_line()
         if not text_lines:
             return False
 
         for text_line_index, text_line in enumerate(text_lines):
-            if text_line_index == 0:
-                line_width = first_width
-            else:
-                line_width = full_width
+            if text_line_index > 0:
                 self.ln()
             new_page = self._render_styled_text_line(
                 text_line,
-                line_width,
                 h=h,
                 border=0,
                 new_x=XPos.WCONT,
                 new_y=YPos.TOP,
-                align=Align.L,
                 fill=False,
                 link=link,
             )
             page_break_triggered = page_break_triggered or new_page
-        # pylint: disable=undefined-loop-variable
         if text_line.trailing_nl:
             # The line renderer can't handle trailing newlines in the text.
             self.ln()
         return page_break_triggered
 
+    @check_page
+    def text_columns(
+        self,
+        text: Optional[str] = None,
+        ncols: int = 1,
+        gutter: float = 10,
+        balance: bool = False,
+        align: Union[Align, str] = "LEFT",
+        line_height: float = 1,
+        l_margin: float = None,
+        r_margin: float = None,
+        print_sh: bool = False,
+        wrapmode: WrapMode = WrapMode.WORD,
+        skip_leading_spaces: bool = False,
+    ):
+        """Establish a layout with multiple columns to fill with text.
+        Args:
+            text (str, optional): A first piece of text to insert.
+            ncols (int, optional): the number of columns to create. (Default: 1).
+            gutter (float, optional): The distance between the columns. (Default: 10).
+            balance: (bool, optional): Specify whether multiple columns should end at approximately
+                the same height, if they don't fill the page. (Default: False)
+            align (Align or str, optional): The alignment of the region. (Default: "LEFT")
```

Comment:
```suggestion
            align (Align or str, optional): The alignment of the region relative to the page. (Default: "LEFT")
```
If I got it wrong why trying clarify this sentence, and it's about the **text alignement**,
that I suggest that this parameter is renamed into `text_align`, similarly to what we did for tables, in order to get rid of this confusing ambiguity.

Reply:
It's actually about the text.
If the tables already use `text_align`, then I better adopt that as well.